### PR TITLE
docs: Prepare for v0.3.2 release

### PR DESCRIPTION
**Pull request by mrnugget**
Preparing 0.3.2 with https://github.com/sourcegraph/scip/pull/217 so I can update it in `sourcegraph/sourcegraph`.

### Test plan

- CI and ran `cargo tests` locally.

### CODE REVIEW

**Review by olafurpg — APPROVED** [review]
LGTM 👍🏻 

Side-comment about the usage of a changelog.md, I'd love to just be able to push a git tag without a PR updating the changelog and bumping up the cargo.toml file.

**Comment by mrnugget** [comment]
go tests failing, taking a look.

**Comment by mrnugget** [comment]
Fixed and updated `Development.md` to add that task.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # ChangeLog
 
+## v0.3.2
+
+SCIP schema:
+
+- Added new `SymbolRole`: `ForwardDefinition`
+  (https://github.com/sourcegraph/scip/pull/217)
+
```
Comment (varungandhi-src) [inline CHANGELOG.md:9]:
```suggestion

scip CLI:

- `scip print` supports turning off colorful output using `--color=false` or
   by setting the environment variable `NO_COLOR=1`.
   (https://github.com/sourcegraph/scip/pull/214)
```

File: Development.md [hunk 2]
```diff
@@ -87,9 +87,8 @@ PACKAGE=MY_PACKAGE_NAME SRC_ACCESS_TOKEN=MY_TOKEN SRC_ENDPOINT=https://sourcegra
 
 ## Release a new version
 
-First, add release notes to the [CHANGELOG](CHANGELOG.md).
-Next, update the version in `cmd/scip/version.txt`
-and `bindings/rust/Cargo.toml`.
+First, add release notes to the [CHANGELOG](CHANGELOG.md). Next, update the
+version in `cmd/scip/version.txt`, `bindings/rust/Cargo.toml` and `docs/CLI.md`
```
Comment (varungandhi-src) [inline Development.md:91]: Sorry/Thanks!

**Review by varungandhi-src — APPROVED** [review]

**Comment by mrnugget** [comment]
https://github.com/sourcegraph/scip/releases/tag/v0.3.2

File: CHANGELOG.md (hunk 1 above)
Comment (mrnugget) [reply CHANGELOG.md:9]: Woops. I'll add it in separate PR, okay?

**Comment by varungandhi-src** [comment]
> I'd love to just be able to push a git tag without a PR updating the changelog and bumping up the cargo.toml file.

Having a hand-written change log seems more useful than having an automatically generated one... And the marginal effort to create a small PR doesn't seem that high.

**Pull request merged**